packages/astro/e2e/lit-component.test.js

```diff
@@ -8,10 +8,6 @@ const test = testFactory({
 // TODO: configure playwright to handle web component APIs
 // https://github.com/microsoft/playwright/issues/14241
 test.describe('Lit components', () => {
-	test.beforeAll(() => {
-		delete globalThis.window;
```

Comment:
No longer needs to shim the window?

Reply:
All of the shims needed for lit are baked into Lit's node build. I just pushed a commit though because we do need to actually re-shim only `customElements.define`because our baked-in shims in ReactiveElement's node build is forgiving and tries to call anything anyone else has shimmed, but we have a small hack we are relying on so we re-shim that